diff --git a/Content.Client/_MACRO/StrangeMoods/Eui/ModelSqlite.cs b/Content.Client/_MACRO/StrangeMoods/Eui/ModelSqlite.cs
deleted file mode 100644
index 3989f82546..0000000000
--- a/Content.Client/_MACRO/StrangeMoods/Eui/ModelSqlite.cs
+++ /dev/null
@@ -1,165 +0,0 @@
-using System;
-using System.Globalization;
-using System.IO;
-using System.Linq;
-using System.Net;
-using System.Text;
-using System.Text.Json;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Diagnostics;
-using Microsoft.EntityFrameworkCore.Infrastructure;
-using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
-using NpgsqlTypes;
-
-namespace Content.Server.Database
-{
-    public sealed class SqliteServerDbContext : ServerDbContext
-    {
-        public SqliteServerDbContext(DbContextOptions<SqliteServerDbContext> options) : base(options)
-        {
-#if USE_SYSTEM_SQLITE
-            SQLitePCL.raw.SetProvider(new SQLitePCL.SQLite3Provider_sqlite3());
-#endif
-        }
-
-        protected override void OnConfiguring(DbContextOptionsBuilder options)
-        {
-            ((IDbContextOptionsBuilderInfrastructure) options).AddOrUpdateExtension(new SnakeCaseExtension());
-
-            options.ConfigureWarnings(x =>
-            {
-                x.Ignore(CoreEventId.ManyServiceProvidersCreatedWarning);
-#if DEBUG
-                // for tests
-                x.Ignore(CoreEventId.SensitiveDataLoggingEnabledWarning);
-#endif
-            });
-
-#if DEBUG
-            options.EnableSensitiveDataLogging();
-#endif
-        }
-
-        protected override void OnModelCreating(ModelBuilder modelBuilder)
-        {
-            base.OnModelCreating(modelBuilder);
-
-            var ipConverter = new ValueConverter<IPAddress, string>(
-                v => v.ToString(),
-                v => IPAddress.Parse(v));
-
-            modelBuilder.Entity<Player>()
-                .Property(p => p.LastSeenAddress)
-                .HasConversion(ipConverter);
-
-            var ipMaskConverter = new ValueConverter<NpgsqlInet, string>(
-                v => InetToString(v.Address, v.Netmask),
-                v => StringToInet(v)
-            );
-
-            modelBuilder
-                .Entity<ServerBan>()
-                .Property(e => e.Address)
-                .HasColumnType("TEXT")
-                .HasConversion(ipMaskConverter);
-
-            modelBuilder
-                .Entity<ServerRoleBan>()
-                .Property(e => e.Address)
-                .HasColumnType("TEXT")
-                .HasConversion(ipMaskConverter);
-
-            var jsonStringConverter = new ValueConverter<JsonDocument, string>(
-                v => JsonDocumentToString(v),
-                v => StringToJsonDocument(v));
-
-            var jsonByteArrayConverter = new ValueConverter<JsonDocument?, byte[]>(
-                v => JsonDocumentToByteArray(v),
-                v => ByteArrayToJsonDocument(v));
-
-            modelBuilder.Entity<AdminLog>()
-                .Property(log => log.Json)
-                .HasConversion(jsonStringConverter);
-
-            modelBuilder.Entity<Profile>()
-                .Property(log => log.Markings)
-                .HasConversion(jsonByteArrayConverter);
-
-            modelBuilder.Entity<Profile>()
-                .Property(log => log.OrganMarkings)
-                .HasConversion(jsonByteArrayConverter);
-
-            // Begin CD - Character Records
-            modelBuilder.Entity<CDModel.CDProfile>()
-                .Property(log => log.CharacterRecords)
-                .HasConversion(jsonByteArrayConverter);
-            // End CD - Character Records
-
-            // EF core can make this automatically unique on sqlite but not psql.
-            modelBuilder.Entity<IPIntelCache>()
-                .HasIndex(p => p.Address)
-                .IsUnique();
-        }
-
-        public override int CountAdminLogs()
-        {
-            return AdminLog.Count();
-        }
-
-        private static string InetToString(IPAddress address, int mask) {
-            if (address.IsIPv4MappedToIPv6)
-            {
-                // Fix IPv6-mapped IPv4 addresses
-                // So that IPv4 addresses are consistent between separate-socket and dual-stack socket modes.
-                address = address.MapToIPv4();
-                mask -= 96;
-            }
-            return $"{address}/{mask}";
-        }
-
-        private static NpgsqlInet StringToInet(string inet) {
-            var idx = inet.IndexOf('/', StringComparison.Ordinal);
-            return new NpgsqlInet(
-                IPAddress.Parse(inet.AsSpan(0, idx)),
-                byte.Parse(inet.AsSpan(idx + 1), provider: CultureInfo.InvariantCulture)
-            );
-        }
-
-        private static string JsonDocumentToString(JsonDocument document)
-        {
-            using var stream = new MemoryStream();
-            using var writer = new Utf8JsonWriter(stream, new JsonWriterOptions {Indented = false});
-
-            document.WriteTo(writer);
-            writer.Flush();
-
-            return Encoding.UTF8.GetString(stream.ToArray());
-        }
-
-        private static JsonDocument StringToJsonDocument(string str)
-        {
-            return JsonDocument.Parse(str);
-        }
-
-        private static byte[] JsonDocumentToByteArray(JsonDocument? document)
-        {
-            if (document == null)
-            {
-                return Array.Empty<byte>();
-            }
-
-            using var stream = new MemoryStream();
-            using var writer = new Utf8JsonWriter(stream, new JsonWriterOptions {Indented = false});
-
-            document.WriteTo(writer);
-            writer.Flush();
-
-            return stream.ToArray();
-        }
-
-        private static JsonDocument ByteArrayToJsonDocument(byte[] str)
-        {
-            return JsonDocument.Parse(str);
-        }
-    }
-}
diff --git a/Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs b/Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs
index 5107adf1a4..1d8cf97c34 100644
--- a/Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs
+++ b/Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs
@@ -6,7 +6,6 @@
 using Content.Server.Electrocution;
 using Content.Server.Explosion.EntitySystems;
 using Content.Server.GhostKick;
-using Content.Client.Nutrition.EntitySystems; // imp edit
 using Content.Server.Physics.Components;
 using Content.Server.Pointing.Components;
 using Content.Server.Polymorph.Systems;
@@ -61,6 +60,7 @@
 using Robust.Shared.Utility;
 using Timer = Robust.Shared.Timing.Timer;
 using Content.Server.Resist; //imp
+using Content.Server.Nutrition.EntitySystems; // imp edit
 
 namespace Content.Server.Administration.Systems;
 
diff --git a/Content.Server/Administration/Systems/AdminVerbSystem.cs b/Content.Server/Administration/Systems/AdminVerbSystem.cs
index 2a4d309e60..a184eb5825 100644
--- a/Content.Server/Administration/Systems/AdminVerbSystem.cs
+++ b/Content.Server/Administration/Systems/AdminVerbSystem.cs
@@ -69,7 +69,7 @@ public sealed partial class AdminVerbSystem : EntitySystem
         [Dependency] private readonly AdminFrozenSystem _freeze = default!;
         [Dependency] private readonly IPlayerManager _playerManager = default!;
         [Dependency] private readonly SiliconLawSystem _siliconLawSystem = default!;
-        [Dependency] private readonly StrangeMoodsSystem _moods = default!; // imp
+        [Dependency] private readonly IPrototypeManager _protoMan = default!; // imp, added due to macro needing protoMan
 
         private readonly Dictionary<ICommonSession, List<EditSolutionsEui>> _openSolutionUis = new();
 
diff --git a/Content.Server/_Impstation/Speech/EntitySystems/ThavenAccentSystem.cs b/Content.Server/_Impstation/Speech/EntitySystems/ThavenAccentSystem.cs
index 2ab9f03892..02ffb832c6 100644
--- a/Content.Server/_Impstation/Speech/EntitySystems/ThavenAccentSystem.cs
+++ b/Content.Server/_Impstation/Speech/EntitySystems/ThavenAccentSystem.cs
@@ -1,5 +1,5 @@
 using System.Text.RegularExpressions;
-using Content.Server._Impstation.Speech.Components;
+using Content.Shared._MACRO.Speech.Components;
 using Content.Server.Speech.EntitySystems;
 using Content.Shared.Speech;
 
diff --git a/Content.Server/_MACRO/Administration/Systems/AdminVerbSystem.MACRO.cs b/Content.Server/_MACRO/Administration/Systems/AdminVerbSystem.MACRO.cs
index de88b8b4e8..933841eadf 100644
--- a/Content.Server/_MACRO/Administration/Systems/AdminVerbSystem.MACRO.cs
+++ b/Content.Server/_MACRO/Administration/Systems/AdminVerbSystem.MACRO.cs
@@ -67,7 +67,7 @@ private void AddMACROTricks(GetVerbsEvent<Verb> args)
                     if (HasComp<StrangeMoodsComponent>(args.Target))
                         return;
 
-                    var ui = new StrangeMoodsInitEui(_moods, EntityManager, ProtoMan, _random, _adminManager, _playerManager, _euiManager, args.User);
+                    var ui = new StrangeMoodsInitEui(_moods, EntityManager, _protoMan, _random, _adminManager, _playerManager, _euiManager, args.User);
                     if (!_playerManager.TryGetSessionByEntity(args.User, out var session))
                         return;
 
